backend/FwLite/MiniLcm/Validators/MultiStringValidator.cs

```diff
@@ -5,8 +5,13 @@ namespace MiniLcm.Validators;
 
 internal static class MultiStringValidator
 {
-    public static IRuleBuilderOptions<T, MultiString> ValidMultiString<T>(this IRuleBuilder<T, MultiString> ruleBuilder)
+    public static IRuleBuilderOptions<T, MultiString> Required<T>(this IRuleBuilder<T, MultiString> ruleBuilder)
     {
-        return ruleBuilder.NotEmpty();
+        return ruleBuilder.NotEmpty().NoEmptyValues();
+    }
+    public static IRuleBuilderOptions<T, MultiString> NoEmptyValues<T>(this IRuleBuilder<T, MultiString> ruleBuilder)
```

Comment:
This forces the front end to remove keys if they become empty. Is that what we want? What if we standardize them when they arrive instead?

Reply:
yeah I was thinking about that too, we could do that with a custom sterilizer which I think would be worth our time for MultiString, though I still think we should validate even if we have code to help avoid the problem